### ISSUE TITLE
Confirm the strategy for deploying the iPad

### ISSUE THREAD

**Issue by huguesf** [cadets-ca/ets#15]
Create new Apple ID for NW and ATL iPads or reuse them.
- Limit of 10 device per AppleID
- Want to use the old app in parallele with new app?
- Sync of the data from old app to new app?

**Comment by kirvanp** [comment]
I do not believe the 10 device limit applies to what we want to do, since it is for the following:

* Turning on iCloud Music Library.
* Downloading past purchases you made in the iTunes Store, App Store, iBooks Store, or Mac App Store.
* Turning on Automatic Downloads.
* Turning on Family Sharing.
* Subscribing to iTunes Match.

However, I have never tested the app over 10 devices as we have 9 gliding sites. 

I do know that the app could be adjusted to not require devices to be on the same AppleID for syncing. Right now, the "Share Database" feature allows people with other AppleID's read-only access. This could, in theory, be adjusted to create a two-way sync.
